libs/barista-components/overlay/src/overlay.ts

```diff
@@ -93,11 +92,16 @@ const DT_OVERLAY_POSITIONS: ConnectedPosition[] = [
  */
 export const DT_OVERLAY_NO_POINTER_CLASS = 'dt-no-pointer';
 
+// TODO: Deprecate this once we have a tooltip component in place
```

Comment:
Have you created a follow-up issue for this?

Reply:
We already have #86  - i'll add more information in the next days there.